pages/form.js

```diff
@@ -1,14 +1,15 @@
 import ChallengeForm from "../components/ChallengeForm";
 import Link from "next/link";
 import React from "react";
+import { RoundButton } from "../components/StyledComponents";
 
-export default function Home() {
+export default function FormPage({ onAddChallenge }) {
   return (
     <main>
-      <ChallengeForm />
-      <h3>
-        <Link href="..">Back to challenges</Link>
-      </h3>
+      <ChallengeForm onAddChallenge={onAddChallenge} />
+      <Link href="/" aria-label="back">
+        <RoundButton>❮</RoundButton>
```

Comment:
The `RoundButton` component is an HTML `button` element, which is unnecessary inside of a Link. I suggest to rather use a styled `<span>` to be more semantic. The rest is great!